src/__test__/run_spec.js

```diff
@@ -7,25 +7,32 @@ const dirname = path.resolve(__dirname, "./__fixtures__");
 
 export default function run_spec(name, options) {
   fs.readdirSync(dirname).forEach(filename => {
-    const source = read(`${dirname}/${filename}`);
-    const output = transform(filename, source, options);
-    test(`${filename}-transform: ${name}`, () => {
-      expect(source + "~".repeat(80) + "\n" + output).toMatchSnapshot(filename);
-    });
+    if (filename === "all.js") {
```

Comment:
This should be removed. Tests should be ran against all fixtures or should be isolated